.github/workflows/release-connect-init.yml

```diff
@@ -0,0 +1,98 @@
+name: "[Release] connect NPM and v9"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit_sha:
+        description: "The commit SHA to checkout"
+        required: true
+        type: string
+      deploymentType:
+        description: "Select the deployment type. (example: canary, stable)"
+        required: true
+        type: choice
+        options:
+          - canary
+          - stable
+
+jobs:
+  # Version should have been bumped by now thanks to ./ci/scripts/connect-release-init-npm.js
+  extract-version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.set-version.outputs.version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # Number of commits to fetch. 0 indicates all history for all branches and tags.
+          fetch-depth: 0
+          # Checkout the specified commit
+          ref: ${{ github.event.inputs.commit_sha }}
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Extract connect version
+        id: set-version
+        run: echo "version=$(node ./ci/scripts/get-connect-version.js)" >> $GITHUB_OUTPUT
+
+  create-push-release-branch:
+    needs: [extract-version]
+    name: "Create release branch for version ${{ needs.extract-version.outputs.version }}"
+    runs-on: ubuntu-latest
+    outputs:
+      branch_name: ${{ steps.push-branch.outputs.branch_name }}
```

Comment:
this + line 74 make `${{ needs.create-push-release-branch.outputs.branch_name }}` available for subsequent jobs, am I right?

Reply:
Yes.